portal/src/graphql/adminapi/schema.graphql

```diff
@@ -1471,6 +1456,21 @@ type SetDisabledStatusPayload {
   user: User!
 }
 
+""""""
+input SetPasswordExpiredInput {
+  """Indicate whether the user's password is expired."""
+  isExpired: Boolean!
```

Comment:
According to the spec, it is called `expired`

Reply:
Updated along with above comments :adore2x: